### DISCUSSION TITLE
End of Life and End of Service of BouncyCastle.Crypto version 1.9.0

### DISCUSSION THREAD

**Discussion by Hemant-700** [bcgit/bc-csharp#450] [Q&A]
Hi,
What is the **End of Life** and **End of Service** of BouncyCastle.Crypto version **1.9.0**

**Comment by dghgit** [comment]
Conventionally we'd regard 1.9.0 as EOL end of 2025, assuming we had people with support contracts using it. We don't currently have any users paying for support of this version though, so at the moment any further work on the 1.9.X series will be discretionary and subject to available resources.
